build-support/bin/generate_user_list.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import pkgutil
+from dataclasses import dataclass
+
+import pystache
+
+"""Generates the custom HTML/CSS block in https://www.pantsbuild.org/docs/who-uses-pants .
```

Comment:
Note that this URL is not currently valid, because the new page doesn't exist in 2.6. But this will be the permanent URL soon enough, so that felt like the right one to put in this docstring.